app/assets/stylesheets/welcome.scss

```diff
@@ -36,3 +37,21 @@ input#original_url {
 .urls-index-header {
   margin-left: 700px;
 }
+
+input#url_original_url {
+  border-radius: 25px;
+  padding: 6px;
+  outline: none;
+}
+
+.btn {
+  border-radius: 25px;
+}
+
+#text_box {
+  margin-top: 50px;
```

Comment:
Properties should be ordered border, border-radius, margin-top, outline, padding